ui/components/app/modular-confirmations/confirmation-row.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ConfirmationRow } from './confirmation-row';
+
+const ConfirmationRowStory = {
+  title: 'Components/App/Modular Confirmations/Row',
```

Comment:
[suggestion] I think we can remove "Modular" from the file name and component names. We plan to modularize more components in the future and keeping it out of the name can help reduce noise

Reply:
Done!